sbcommon.h

```diff
@@ -1586,7 +1593,31 @@ class AppSettings
 				}
 				if (strTemp2.CompareNoCase(_T("EnableMediaAccessByApproval")) == 0)
 				{
-					EnableMediaAccessByApproval = strTemp3 == _T("1") ? TRUE : FALSE;
+					if (strTemp3 == _T("1"))
+					{
+						// EnableMediaAccessByApproval was deprecated.
+						// 1: means manual approval, so migrate EnableMediaAccessPermission = 1
+						EnableMediaAccessPermission = AppSettings::MediaAccessPermission::MANUAL_MEDIA_APPROVAL;

```

Comment:
Memo for me:

If `EnableMediaAccessByApproval` exists, it is migrated to `EnableMediaAccessPermission` and saved to the config file as `EnableMediaAccessPermission`  after migration.
If both `EnableMediaAccessByApproval`  and `EnableMediaAccessPermission` exist in the config file, the later one in the config file will take effect.

Reply:
Added note to commit message.

Reply:
Thanks!